utils/src/main/java/org/corfudb/utils/lock/LockClient.java

```diff
@@ -129,6 +140,24 @@ public void registerInterest(@NonNull String lockGroup, @NonNull String lockName
         monitorLocks();
     }
 
+    /**
+     * Enforce a transition to NoLeaseState and stop monitoring.
+     **/
+    public void suspendInterest() {
```

Comment:
do we need this functionality or is it only for testing?

Reply:
Now for testing purpose. It is possible in future if we detect a slow node that holding the lock, we can enforce it to release the lock.